modules/admin-ui/src/main/resources/public/org/opencastproject/adminui/languages/lang-en_US.json

```diff
@@ -891,9 +891,11 @@
          },
          "THEME": {
            "CAPTION": "Theme",
-           "DESCRIPTION": "",
+           "DESCRIPTION": {
+             "TEXT": "You can optionally use themes to brand your videos. Themes may define bumper and trailer videos, watermarks and title slides including metadata. The brandings defined by the theme will be applied when publishing events of this series."
```

Comment:
This text is not correct. The themes can be applied at any time during a workflow execution or not at all. It is a configurable operation. It certainly may not only happen when publishing material.